src/Frontend/reducers/locale/localeReducer.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * This file is part of the Sententiaregum project.
+ *
+ * (c) Maximilian Bosch <maximilian.bosch.27@gmail.com>
+ * (c) Ben Bieler <benjaminbieler2014@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import { GET_LOCALES, CHANGE_LOCALE } from '../../constants/Locale';
+import initialState                 from '../../config/initialState';
+
+const localeReducer = (state = initialState, action) => {
```

Comment:
the `initialState` contains the entire state tree (not only the state sub-tree of the locale). Therefore it should be `state = /* default state of the locale */` (look at the `menuReducer`, we did the same there)